src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/FormError.java

```diff
@@ -0,0 +1,14 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class FormError {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)